### ISSUE TITLE
Compile error, Variant form 'cnV' deprecated for base form '\prop_gput:cnn'

### ISSUE THREAD

**Issue by hebaek** [cgnieder/leadsheets#17]
When compiling in pdflatex and xelatex, I get the following compile errors:

```
! LaTeX3 Error: Variant form 'cnV' deprecated for base form '\prop_gput:cnn'.
(LaTeX3)        One should not change an argument from type 'c' to type 'c':
(LaTeX3)        base form is already a variant.

Type <return> to continue.
 ...                                              
                                                  
l.45 \cs_generate_variant:Nn \prop_gput:cnn {cnV}

```
It seems to be in leadsheets.library.properties.code.tex.

**Comment by konnykonny50** [comment]
Hello,
i get the same error now. Probably after updating some LaTeX packages. Have you solved the problem already?

**Comment by hebaek** [comment]
Hello!

No, I have not solved it, it involves a bit too much code fiddeling for my skills...
I am using TexLive on mac, by the way.

**Comment by konnykonny50** [comment]
Found a fix that works for me at least so far. But I am not realy confident about what I did there. Change the following line in leadsheets.library.properties.code.tex:45 from
`\cs_generate_variant:Nn \prop_gput:cnn {cnV}`
to
`\cs_generate_variant:Nn \prop_gput:cnV {cnV}`

**Comment by hebaek** [comment]
Nice!

I tried another solution, which also may or may not work (it works for the very limited stuff I'm doing right now): I deleted the same line of code you changed. But like you, I am not confident this is the right solution, it just happens to work for me right now.

I wish @cgnieder or someone else with a deeper understanding of the code would look into this, but it seems this project is somewhat dormant.
